### ISSUE TITLE
ExqUi doesn't work in chrome - mime type

### ISSUE THREAD

**Issue by azranel** [akira/exq_ui#54]
https://monosnap.com/file/KJAiY6SQGorG2TpV0VWJEPFKlWNGkT - says all I think

Will try to fix it later 😄

**Comment by nicholasjhenry** [comment]
@azranel what this happening in `dev`? I had the same issue, but it was because my namespace was misconfigured and I was receiving an HTML page instead of the JS. Once I corrected my namespace this worked fine in Chrome.

**Comment by alkx** [comment]
@nicholasjhenry do you remember what was misconfigured with your namespace? I'm seeing the same issue.

**Comment by nicholasjhenry** [comment]
Yeah, I wish I was a little more explicit about that. I would verify the following:

- `plug ExqUi.RouterPlug, namespace: "exq"` in your router
- `config :exq_ui,
  web_port: 4040,
  web_namespace: "",
  server: true`

It's possible I had `web_namespace: "exq"` by mistake.

**Comment by ananthakumaran** [comment]
Closing this, please re-open if the issue still happens on the latest release.
